tests/dsdl/_compiler.py

```diff
@@ -1,13 +1,16 @@
 # Copyright (c) 2020 OpenCyphal
 # This software is distributed under the terms of the MIT License.
 # Author: Pavel Kirienko <pavel@opencyphal.org>
-
+import sys
 import typing
 import logging
 import pathlib
 import tempfile
 import pytest
 import pycyphal.dsdl
+from pycyphal.dsdl._import_hook import DsdlMetaFinder
```

Comment:
Protected name imports better be local. Please move this one into the test function scope.